tutorial/cloudshell_tutorial.md

```diff
@@ -143,263 +211,296 @@ rerun this command to reinitialize your working directory. If you forget, other
+      + project                    = "rln-gcp-test-01"
+      + service                    = "compute.googleapis.com"
     }
-  # google_project_service.service["oslogin.googleapis.com"] will be created
-  + resource "google_project_service" "service" {
-      + disable_on_destroy = true
-      + id                 = (known after apply)
-      + project            = "just-center-247116"
-      + service            = "oslogin.googleapis.com"
+
+  # module.project_services.google_project_service.project_services[1] will be created
+  + resource "google_project_service" "project_services" {
+      + disable_dependent_services = false
+      + disable_on_destroy         = false
+      + id                         = (known after apply)
+      + project                    = "rln-gcp-test-01"
+      + service                    = "oslogin.googleapis.com"
     }
+
 Plan: 2 to add, 0 to change, 0 to destroy.
+
 Do you want to perform these actions?
   Terraform will perform the actions described above.
   Only 'yes' will be accepted to approve.
 
-
+  Enter a value:
 ```
 
-Respond with `yes`. When you do, you should see further output like this:
+Respond with a `yes` to have Terraform enable these services.
 
 ```raw
-...
-Do you want to perform these actions?
-  Terraform will perform the actions described above.
-  Only 'yes' will be accepted to approve.
   Enter a value: yes
-google_project_service.service["oslogin.googleapis.com"]: Creating...
-google_project_service.service["compute.googleapis.com"]: Creating...
-google_project_service.service["oslogin.googleapis.com"]: Still creating... [10s elapsed]
-google_project_service.service["compute.googleapis.com"]: Still creating... [10s elapsed]
-google_project_service.service["oslogin.googleapis.com"]: Creation complete after 15s [id=just-center-247116/oslogin.googleapis.com]
-google_project_service.service["compute.googleapis.com"]: Creation complete after 15s [id=just-center-247116/compute.googleapis.com]
-Apply complete! Resources: 2 added, 0 changed, 0 destroyed.
-```
 
-Adding this resource is the equivalent of navigating to the appropriate page in
-the [api
-library](https://console.developers.google.com/apis/library/compute.googleapis.com)
-and enabling the services there.
+module.project_services.google_project_service.project_services[0]: Creating...
+module.project_services.google_project_service.project_services[1]: Creating...
+module.project_services.google_project_service.project_services[1]: Creation complete after 3s [id=rln-gcp-test-01/oslogin.googleapis.com]
+module.project_services.google_project_service.project_services[0]: Creation complete after 3s [id=rln-gcp-test-01/compute.googleapis.com]
 
-You may also have noticed that we have to repeat the project ID. Don't worry, we'll fix that soon when we learn about _variables_.
+Apply complete! Resources: 2 added, 0 changed, 0 destroyed.
+```
 
-## Provisioning Resources
+## Create infrastructure
```

Comment:
Should this be combined with the previous section so that students only need to do the apply once?

Reply:
It can take a few seconds to enable the project_services, and so I sometimes get an error when trying to apply both the project_services and the network at the same time. Splitting the applies up works around the issue.